File: aoj/volume100/csharp/A10025.cs
using System;

class AOJ
{
	static void Main()
	{
		var input = Console.ReadLine().Split();

		var a = double.Parse(input[0]);
		var b = double.Parse(input[1]);
		var deg = double.Parse(input[2]);
		var rad = deg * Math.PI / 180;
		var c = Math.Sqrt(Math.Pow(a, 2) + Math.Pow(b, 2) - 2*a*b*Math.Cos(rad));

		var area = a*b*Math.Sin(rad) / 2.0;
		var perimeter = a + b + c;
		var h = 2*area / a;
		Console.WriteLine(area.ToString("f6"));
		Console.WriteLine(perimeter.ToString("f6"));
		Console.WriteLine(h.ToString("f6"));
	}
}

File: aoj/volume100/python3/A10025.py
import math

a, b, deg = map(float, input().split())

rad = deg * math.pi / 180
c = (a**2 + b**2 - 2*a*b*math.cos(rad)) ** 0.5

area = a*b*math.sin(rad) / 2.0
perimeter = a + b + c
h = 2*area / a

print("%.6f\n%.6f\n%.6f\n" % (area, perimeter, h))
